docs/source/markdown/podman-volume-create.1.md

```diff
@@ -129,6 +129,77 @@ All containers are assigned larger project IDs (e.g. >= 100000).
 All volume assigned project IDs larger project IDs starting with 200000.
 This prevents xfs_quota management conflicts with containers/storage.
 
+## MOUNT EXAMPLES
+
+podman volume create allows the `type`, `device`, and `o` options to be passed to `mount(8)` when using the `local` driver.
+
+## [s3fs-fuse](https://github.com/s3fs-fuse/s3fs-fuse)
+
+[s3fs-fuse](https://github.com/s3fs-fuse/s3fs-fuse) or just `s3fs`, is a [fuse](https://github.com/libfuse/libfuse) filesystem that allows s3 prefixes to be mounted as filesystem mounts.
+
+**Installing:**
+```shell
+$ doas dnf install s3fs-fuse
+```
+
+**Simple usage:**
+```shell
+$ s3fs --help
+$ s3fs -o use_xattr,endpoint=aq-central-1 bucket:/prefix /mnt
+```
+
+**Equivalent through `mount(8)`**
+```shell
+$ mount -t fuse.s3fs -o use_xattr,endpoint=aq-central-1 bucket:/prefix /mnt
+```
+
+**Equivalent through `podman volume create`**
+```shell
+$ podman volume create s3fs-fuse-volume -o type=fuse.s3fs -o device=bucket:/prefix -o o=use_xattr,endpoint=aq-central-1
+```
+
+**The volume can then be mounted in a container with**
+```shell
+$ podman run -v s3fs-fuse-volume:/s3:z --rm -it fedora:latest
+```
+
+Please see the available [options](https://github.com/s3fs-fuse/s3fs-fuse/wiki/Fuse-Over-Amazon#options) on their wiki.
+
+### Using with other container users
+
+The above example works because the volume is mounted as the host user and inside the container `root` is mapped to the user in the host.
+
+If the mount is accessed by a different user inside the container, a "Permission denied" error will be raised. Even though that user should be mapped to one of the caller's `subuid`.
```

Comment:
Sentence fragment. After thinking about it, I recommend just removing the entire "Even though ... subuid" fragment, it doesn't add anything. Just the first sentence ("If the mount ... will be raised") is sufficient.

Reply:
I'd argue that it gets the reader to think about how a `subid` interacts with the host.
Hindsight, it's obvious, but when I was figuring this out it wasn't so. Working on this gave me a "concrete" understanding of what user is actually "inside" / "outside" the container.
The hope would be to help someone else "get-it"

But I agree this sentence isn't great, and the value of the above is questionable at best.
Removed.